src/main/java/uk/gov/hmcts/darts/common/entity/Courthouse.java

```diff
@@ -21,34 +20,21 @@ public class Courthouse {
 
     @Id
     @Column(name = "moj_crt_id")
-    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    @GeneratedValue(strategy = GenerationType.SEQUENCE, generator = "courthouse_gen")
+    @SequenceGenerator(name = "courthouse_gen", sequenceName = "moj_crt_seq", allocationSize = 1)
     private Integer id;
 
-    @Column(name = "r_courthouse_object_id", length = 16)
-    private String legacyObjectId;
+    @Column(name = "courthouse_code", unique = true)
+    private short code;
```

Comment:
If this column is NULLable - then this type should probably be an `Short` instead?
Otherwise it would default to 0?